test/modules/TestResultCoordinator/report/SimpleTestResults.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Storage;
+
+    class SimpleTestResults<T> : ITestResults<T>
```

Comment:
```suggestion
    class TestResultSource<T> : ITestResultSource<T>
```

Reply:
As discussed, renamed to StoreTestResultCollection